python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -783,7 +797,7 @@ def __init__(cls, name, bases, this_dict):
         this_schema._pluginclass = cls
 
         # store generated proxy class
-        cls._proxyClass = proxyClass
+        setattr(cls, '_proxyClass', proxyClass)
```

Comment:
This was just a minor change which I made to highlight that this probably shouldn't be in Objects.py in the long-run.